simulariumio/trajectory_converter.py

```diff
@@ -122,19 +122,19 @@ def _read_trajectory_data(input_data: TrajectoryData) -> Dict[str, Any]:
             "version": 1,
             "msgType": 1,
             "bundleStart": 0,
-            "bundleSize": totalSteps,
+            "bundleSize": total_steps,
         }
         if input_data.agent_data.subpoints is not None:
             spatialData[
                 "bundleData"
             ] = TrajectoryConverter._get_spatial_bundle_data_subpoints(
-                input_data.agent_data
+                input_data.agent_data, type_ids
             )
         else:
             spatialData[
                 "bundleData"
             ] = TrajectoryConverter._get_spatial_bundle_data_no_subpoints(
-                input_data.agent_data
+                input_data.agent_data, type_ids
```

Comment:
Everywhere else you're removing type_ids but how come you're adding it here?

Reply:
Oh I think I got it... we still want to write type_ids to the output simularium file so this is doing that?

Reply:
yeah this is basically the one place they are generated now. I realized I can also remove it in TrajectoryConverter.add_number_of_agents_plot() too